packages/studio-plugin/src/createStudioPlugin.ts

```diff
@@ -33,9 +33,6 @@ export default async function createStudioPlugin(
   );
   const pathToUserProjectRoot = getProjectRoot(cliArgs);
   const studioConfig = await getStudioConfig(pathToUserProjectRoot, cliArgs);
-  const pagesDevPortPromise = studioConfig.isPagesJSRepo
```

Comment:
ahh, gotcha.  I moved it down since we get `isWithinCBD` by checking `!!process.env.YEXT_CBD_BRANCH` in the ParsingOrchestrator, and it would have been nice to only check `!!process.env.YEXT_CBD_BRANCH` in once spot.  But yeah, that makes sense, I'll move it back up!

Reply:
https://yexttest.atlassian.net/browse/SLAP-2903

Reply:
You can still run them in parallel here by starting createDevServer at the too and awaiting it at the bottom like before

then the item you made can be a refactoring item to fully separate async deps into a Promise.all call